src/runtime_src/core/pcie/tools/xbmgmt/cmd_config.cpp

```diff
@@ -155,10 +155,23 @@ static void showDevConf(std::shared_ptr<pcidev::pci_device>& dev)
     if (!errmsg.empty()) {
         std::cout << "can't read security level from " << dev->sysfs_name <<
             " : " << errmsg << std::endl;
-        return;
+        goto clk_scale;
```

Comment:
i don't believe using 'goto' is a proper way of writing c++ code. please, at least, create functions to show specific configurations and call them here.